doc/sphinx/source/recipes/recipe_cvdp.rst

```diff
@@ -0,0 +1,19 @@
+CVDP Package
+============
+
+About
+-----
+
+The NCAR Climate Variability Package (CVDP) is a stand alone ncl based application for the analysis of climate variability in models and observations, see [1] and [2]. The CVDP package is also distributed with the ESMValTool.
+
+Requirements
+------------
+
++ nco (optional for creating netcdf files)
```

Comment:
Maybe you can add the nco package to environment.yml and meta.yml, so it gets installed.

Reply:
I had to remove `nco` again. It broke the environment and made all the tests fail.